omero/sysadmins/system-requirements.txt

```diff
@@ -75,7 +75,7 @@ the user's HOME directory in :file:`$HOME/omero/tmp`,
 :file:`$HOME\\omero\\sessions`). If your home
```

Comment:
Alternatively since this info is also in http://www.openmicroscopy.org/site/support/omero5.1-staging/sysadmins/troubleshooting.html#performance-issues-with-the-clients you could remove the duplication and add a link instead.